### ISSUE TITLE
Alt text

### ISSUE THREAD

**Issue by farishkash** [AWitam/prj-rev-bwfs-tea-cozy#3]
So your images should have a descriptive alt text.  Reason being is that the visually impaired use screen readers.  The screen readers read the alt text to provide them a description of the image.
